app/connectors/ArrivalMovementConnector.scala

```diff
@@ -37,12 +37,26 @@ class ArrivalMovementConnector @Inject() (config: FrontendAppConfig, http: HttpC
 
   private val channel: String = "web"
 
-  def getArrivals()(implicit hc: HeaderCarrier): Future[Option[Arrivals]] = {
+  private def makeQueryParam(param: String, valueParam: Option[String]): Seq[(String, String)] =
+    valueParam.fold(Seq.empty[(String, String)])(
+      value => Seq((param, value))
+    )
```

Comment:
You could take this further and remove the need to ++ by adding a case class, looks good as it is though:

`
case class QueryParam(param: String, valueParam: Option[String])

  private def makeQueryParam(queryParam: QueryParam*): Seq[(String, String)] = {

    queryParam.foldLeft(Seq.empty[(String, String)]){
      (x, y) =>
        y.valueParam.fold(Seq.empty[(String, String)])(
          value => Seq((y.param, value))
        ) ++ x
    }
    
    makeQueryParam(
      QueryParam("mrn", Some("mrn")),
      QueryParam("etc", None)
    )


`